fastai/sgdr.py

```diff
@@ -122,6 +122,10 @@ def plot(self, n_skip=10, n_skip_end=5):
 
 
 class CosAnneal(LR_Updater):
+    """
+    cosine annilation. 
+    see paper: https://openreview.net/pdf?id=BJYwwY9ll
```

Comment:
annealing :)
annilation almost sounds like annihilation, which generally brings images of large nuclear bombs. Think not generally in our business